app/models/parent_object.rb

```diff
@@ -98,6 +105,8 @@ def ladybird_json=(lb_record)
     self.barcode = lb_record["orbisBarcode"]
     self.aspace_uri = lb_record["archiveSpaceUri"]
     self.visibility = lb_record["itemPermission"]
+    self.extent_digitization = lb_record["extentOfDigitization"][0]
```

Comment:
If we're getting this out of an array, I think it's more ruby-ish to use `.first` instead of the index. Same with next row. (this is just a style preference thing, not right / wrong)